doc/examples/filters/plot_deconvolution.py

```diff
@@ -1,55 +1,118 @@
+#!/usr/bin/python
```

Comment:
Is this used in any of the other examples?  If so, should probably be `/usr/bin/env python`.

Reply:
*probably ➡️ definitely! However I'm happy to get this in as-is and fix later. But currently we need a rebase or cmd-line merge.

Reply:
Oh and actually it should just be removed — it's not used anywhere else.